algorithms/CSharp/src/Strings/character-limit.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Linq;
+
+//Time Complexity: O(N)
+namespace Algorithms.Strings
+{
+    public class TextHelper
+    {
+        public static void Main()
+        {
+            Console.WriteLine("Please enter a string");
+            string input = Console.ReadLine();
+
+            while (input == "")
+            {
+                Console.WriteLine("Please enter a string");
+                input = Console.ReadLine();
+            }
+
+            Console.WriteLine("Please enter the number of characters you want to show");
+            string input2 = Console.ReadLine();
+
+            while (!input2.All(char.IsDigit) || input2.All(char.IsWhiteSpace))
+            {
+                Console.WriteLine("The number you have entered is invalid. Please try again:");
+                input2 = Console.ReadLine();
+            }
+
+            int num = Convert.ToInt32(input2);
+            Console.WriteLine(Character_Limit(input, num));
```

Comment:
Rename the method name
```suggestion
            Console.WriteLine(CharacterLimit(input, num));
```